### PR TITLE
Platformify the flight manual

### PR DESCRIPTION
**Pull request by lee-dohm**
Updating all pages to the new standards in the CONTRIBUTING document.

### CODE REVIEW

File: content/using-atom/sections/atom-packages.md [hunk 1]
```diff
@@ -62,26 +66,26 @@ For example `apm install emmet@0.1.5` installs the `0.1.5` release of the [Emmet
...
-├── 1224 downloads
-└── 9 stars
-
-Run `apm install git-grep` to install this package.
+> git-grep
+> ├── 0.7.0
+> ├── git://github.com/mizchi/atom-git-grep
+> ├── `git grep` in atom editor
+> ├── 1224 downloads
+> └── 9 stars>
```
Comment (50Wliu) [inline content/using-atom/sections/atom-packages.md]: Extra >

File: content/getting-started/sections/atom-basics.md [hunk 2]
```diff
@@ -120,46 +123,60 @@ This is a great tool if you're used to the terminal or you work from the termina
...
+
+You can also search through only the files currently opened (rather than every file in your project) with <kbd class="platform-mac">Cmd+B</kbd><kbd class="platform-windows platform-linux">Ctrl+B</kbd>. This searches through your "buffers" or open files. You can also limit this fuzzy search with <kbd class="platform-mac">Cmd+Shift+B</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+B</kbd>, which searches only through the files which are new or have been modified since your last Git commit.
+
+The fuzzy finder uses both the `core.ignoredNames`, `fuzzy-finder.ignoredNames` and `core.excludeVCSIgnoredPaths` configuration settings to filter out files and folders that will not be shown. If you have a project with tons of files you don't want it to search through, you can add patterns or paths to either of these config settings or your [standard `.gitignore` files](https://git-scm.com/docs/gitignore). We'll learn more about config settings in [Global Configuration Settings](/using-atom/sections/basic-customization/#global-configuration-settings), but for now you can easily set these in the Settings View under Core Settings.
```
Comment (50Wliu) [inline content/getting-started/sections/atom-basics.md]: There are three settings listed, so 'both' doesn't make sense.

File: content/using-atom/sections/atom-packages.md (hunk 1 above)
Comment (lee-dohm) [inline content/using-atom/sections/atom-packages.md]: That's intentional:

https://github.com/gjtorikian/extended-markdown-filter#command-line-highlighting

File: content/getting-started/sections/atom-basics.md (hunk 2 above)
Comment (lee-dohm) [inline content/getting-started/sections/atom-basics.md]: Fixed. Thanks!

**Comment by mnquintana** [comment]
YAY

**Comment by lee-dohm** [comment]
Merging this for now because I'm running out of :dash: and I don't want these updates to wait on me getting another good :boom: in.

File: content/using-atom/sections/atom-packages.md (hunk 1 above)
Comment (jerone) [inline content/using-atom/sections/atom-packages.md]: Also for the second `>` just after `stars`?

**Pull request merged**